src/modules/open-ai/api/openAi.ts

```diff
@@ -83,16 +82,8 @@ export async function alterGeneratedImg(
       deleteFile(imageData.fileName!);
       return response?.data;
     } else {
-      ctx.reply(imageData.error).catch((e) => {
-        if (e instanceof GrammyError) {
-          logger.error(
-            `Error when sending message "Error handling your request" - ${e.error_code} - ${e.description}`
-          );
-        } else {
-          logger.error(
-            `Error when sending message "Error handling your request", ${e.toString()} `
-          );
-        }
+      await ctx.reply(imageData.error).catch((e) => {
+        throw e;
       });
```

Comment:
here is no need to use throw e;, just remove the catch block completely, this is applicable to the code below